src/main.js

```diff
@@ -1,7 +1,9 @@
-import "./assets/app.scss";
 import { createApp, h } from "vue";
 import App from "./App.vue";
 
+import "@fortawesome/fontawesome-free/css/all.css";
```

Comment:
Why would you do that ? it's already imported in base, integrated to the CSS layer system. I don't understand :thinking: